pages.zh/common/gpg-zip.md

```diff
@@ -0,0 +1,16 @@
+# gpg-zip
+
+> 使用 GPG 加密存档中的文件和目录。
+
+- 使用密码将一个目录加密为 archive.gpg 。
```

Comment:
There are periods at the end of these example descriptions&mdash;please replace them with colons